tests/framework/framework.go

```diff
@@ -76,34 +80,42 @@ func init() {
 // NewFramework makes a new framework and sets up the global BeforeEach/AfterEach's.
 // Test run-time flags are parsed and added to the Framework struct.
 func NewFramework(prefix string) *Framework {
+	var err error
+
 	f := &Framework{
 		NsPrefix: prefix,
 	}
 
 	// handle run-time flags
 	if !flag.Parsed() {
 		flag.Parse()
-		// report flags values passed to test binary
-		fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
-		flag.Visit(func(f *flag.Flag) {
-			fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
-		})
-		fmt.Fprintf(GinkgoWriter, "**\n")
-		if kubectlPath != nil {
-			f.KubectlPath = *kubectlPath
-		}
-		if ocPath != nil {
-			f.OcPath = *ocPath
-		}
-		if cdiInstallNs != nil {
-			f.CdiInstallNs = *cdiInstallNs
-		}
-		if kubeConfig != nil {
-			f.KubeConfig = *kubeConfig
-		}
-		if master != nil {
-			f.Master = *master
-		}
+	}
+	// report flags values passed to test binary
+	fmt.Fprintf(GinkgoWriter, "** Test flags:\n")
+	flag.Visit(func(f *flag.Flag) {
+		fmt.Fprintf(GinkgoWriter, "   %s = %q\n", f.Name, f.Value.String())
+	})
+	fmt.Fprintf(GinkgoWriter, "**\n")
+	if kubectlPath != nil {
+		f.KubectlPath = *kubectlPath
+	}
+	if ocPath != nil {
+		f.OcPath = *ocPath
+	}
+	if cdiInstallNs != nil {
+		f.CdiInstallNs = *cdiInstallNs
+	}
+	if kubeConfig != nil {
+		f.KubeConfig = *kubeConfig
+	}
+	if master != nil {
+		f.Master = *master
+	}
+
+	f.RestConfig, err = f.LoadConfig()
+	if err != nil {
+		// Can't use Expect here due to fail handler not being registered at this point
+		fmt.Fprintf(GinkgoWriter, "ERROR, unable to load RestConfig")
```

Comment:
This 
Probably should include the cdi _errors_ import so that err can be wrapped and a stack trace is automated when the err is formatted. In this case, I'd write something like:
```

```